docs/writing/gc_content.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python2
```

Comment:
Actually, this made me do some research.. can OS X people confirm whether python2 is on your PATH?